Classes/InfoStop.cpp

```diff
@@ -122,22 +122,24 @@ void InfoStop::update(bool hit)
             planet->setScale(0.65f);
             planet->setPosition(Vec2(300, 260));
             bgSprite->addChild(planet);
-			auto playAudio = CallFunc::create([this, i]() {
-
-				auto audio = SimpleAudioEngine::getInstance();
-				audio->playEffect(("sfx/" + langCode + "/digit_" + std::to_string(i+1) + ".wav").c_str());
-
-			});
-			allActions.pushBack(playAudio);
+            auto playAudio = CallFunc::create([this, i]() {
+                
+                auto audio = SimpleAudioEngine::getInstance();
+                audio->playEffect(("sfx/" + langCode + "/digit_" + std::to_string(i+1) + ".wav").c_str());
+            });
+            allActions.pushBack(playAudio);
             auto move = MoveBy::create(0.75f, Vec2(0,-(160 - 15*i)));
             auto targetedAction = TargetedAction::create(planet, move);
             allActions.pushBack(targetedAction);
             i++;
         }
-        
+        string greaterOrSmaller = "/is_greater_than.wav";
```

Comment:
@protito To achieve some variation, can we randomly select either `is_greater_than.wav` or `is_larger_than.wav`?